src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -122,6 +122,12 @@ enum cmacommand {
     CMA_SIZE,
 };
 
+enum cu_stat {
+    CU_USAGE = 0x0,
+    CU_ADDR,
+    CU_STATUS,
+};
```

Comment:
When we introduce new enums in C++ code, it would be good to make them enum classes with lower case enumeration variables, e.g.
```
enum class cu_stat : unsigned short { 
  usage = 0, 
  addr, 
  stat
 };
```
Enum classes scopes the enumeration variables and there is zero risk of perprocessor interfering provided you use lower case enum variables and no one has #defined a lower case symbolic name.  It also means that you can use short  symbolic to the point names of the enum variables, since these can only be used when scoped with the class name, for example:
`if (kind == cu_stat::usage) ...`